### PR TITLE
[476810] [linking] method unreachable from Xtend

### PR DESCRIPTION
**Pull request by JanKoehnlein**
Generate stubs for nested types as well

Change-Id: Idd1a071bfcb1790a621515e1ed038e0faf9eba71

### CODE REVIEW

**Comment by eclipsewebmaster** [comment]
Issue tracker reference:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=476810

**Comment by JanKoehnlein** [comment]
Jenkins xtext-ide-tests-master-latest

**Comment by JanKoehnlein** [comment]
Jenkins xtext-ide-tests-master-latest

File: tests/org.eclipse.xtend.ide.tests/xtend-gen/org/eclipse/xtend/ide/tests/common/types/access/jdt/WorkingCopyOwnerProviderTest.java [hunk 1]
```diff
@@ -102,7 +102,9 @@ public void testWorkingCopyOwber_02() {
       StringConcatenation _builder_1 = new StringConcatenation();
       _builder_1.append("package foo;");
       _builder_1.newLine();
-      _builder_1.append("public class MyClass{}");
+      _builder_1.append("public static class MyClass{");
```
Comment (szarnekow) [inline tests/org.eclipse.xtend.ide.tests/xtend-gen/org/eclipse/xtend/ide/tests/common/types/access/jdt/WorkingCopyOwnerProviderTest.java]: This is invalid java. The stub generator should only add the static modifier for the nested types.

**Pull request merged**